robot.py

```diff
@@ -7,6 +10,21 @@ def robotInit(self):
     This function is called upon program startup and
     should be used for any initialization code.
     """
+    if wpilib.RobotBase.isReal():  # getDeployData() returns None during simulation
```

Comment:
Recommend you put this into a new myRobot function that gets called by robotInit. It will make the init easier to read and possibly troubleshoot as it gets populated with additional code going forward.

Reply:
I moved the code to a file named `shuffleboard.py`. Might change the name of the file if it's too confusing (there's also a `wpilib.shuffleboard` import).